GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Employer.cshtml

```diff
@@ -307,6 +334,55 @@
                 }
             });
         }());
+        
+        var parentForSelect = document.getElementById("filteringArea");
+        
+        var selectLabel = document.createElement("label");
+        selectLabel.className = "govuk-label";
+        selectLabel.for = "yearFilter";
+        selectLabel.textContent = "Filter for a reporting period";
+        
+        var elements = document.getElementsByClassName("yearTextForFiltering");
+        
+        var selectList = document.createElement("select");
+        selectList.className = "govuk-select govuk-!-margin-bottom-3";
+        selectList.id = "filterYearSelect";
+        selectList.addEventListener("change",function (e){
+            for (var i = 0; i < elements.length; i++){
+                var tableRow = elements[i];
+                while (tableRow.tagName.toLowerCase() !== "tr"){
+                    tableRow = tableRow.parentElement;    
+                }

```

Comment:
Do we need to support IE? If not, you can simplify this to `var tableRow = elements[i].closest("tr")` ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest))

Reply:
@LadunOmideyiSoftwire the first bit is probably a question for you.

Reply:
Firstly, this wasn't me intentionally supporting IE, I was just unaware of that option. But I would presume IE should be supported.

Reply:
Yeah I didn't think this was a deliberate choice for IE compatibility, just wanted to make sure we didn't need to support it before I suggested adding a feature that isn't in IE.